File: baekjoon/bronze/level1/test_2309/Main.java
package bronze.level1.test_2309;

import java.io.*;
import java.util.Arrays;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        int sum = 0;
        int n = 9;
        int[] arr = new int[n];

        for (int i = 0; i < n; i++) {
            arr[i] = Integer.parseInt(br.readLine());
            sum += arr[i];
        }

        Arrays.sort(arr);
        for (int i = 0; i < n; i++) {
            for (int j = i + 1; j < n; j++) {
                if (sum - (arr[i] + arr[j]) == 100) {
                    for (int k = 0; k < n; k++) {
                        if (k != i && k != j) {
                            bw.write(String.valueOf(arr[k]));
                            bw.newLine();
                        }
                    }
                    bw.close();
                    return;
                }
            }
        }
    }
}

File: baekjoon/bronze/level1/test_2309/kotlin/Main.kt
package baekjoon.bronze.level1.test_2309.kotlin

fun main() {
    val arr = IntArray(9)
    var sum = 0

    for (i in 0..8) {
        val value = readln().toInt()
        arr[i] = value
        sum += value
    }
    arr.sort()

    var left = 0
    var right = 8
    while (sum - 100 != arr[left] + arr[right]) {
        if (sum - 100 > arr[left] + arr[right]) {
            left++
        } else {
            right--
        }
    }

    for (i in 0..8) {
        if (i != left && i != right) {
            println(arr[i])
        }
    }
}
